main/writer-ctags.c

```diff
@@ -34,17 +34,23 @@ static int writeCtagsPtagEntry (tagWriter *writer CTAGS_ATTR_UNUSED,
 								void *clientData);
 static bool treatFieldAsFixed (int fieldType);
 
+#ifdef WIN32
+static enum filenameSepOp overrideFilenameSeparator (enum filenameSepOp currentSetting);
+#endif	/* WIN32 */
+
 struct rejection {
 	bool rejectionInThisInput;
 };
 
 tagWriter uCtagsWriter = {
 	.writeEntry = writeCtagsEntry,
 	.writePtagEntry = writeCtagsPtagEntry,
-	.preWriteEntry = NULL,
```

Comment:
Did you intentionally remove this?

Reply:
No. I should not remove this line. I will fix this.